tests/test_deserialization.py

```diff
@@ -349,6 +349,8 @@ def test_boolean_field_deserialization_with_custom_falsy_values(self):
             42,
             '',
             [],
+            dt.datetime.now().strftime('%H:%M:%S %Y-%m-%d'),
```

Comment:
I think this datetime string was rejected before your change as well. It is not the use of `isoparse` that makes it rejected, but the regex. See [my comment](https://github.com/marshmallow-code/marshmallow/issues/899#issuecomment-418042719) and my proposal to also uses Django regexes for date and time.